go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/algorand/conduit
 go 1.17
 
 require (
-	github.com/algorand/go-algorand-sdk/v2 v2.0.0-20230324200319-055c8d2b174a
+	github.com/algorand/go-algorand-sdk/v2 v2.0.0-20230511222239-d0007f96cf95
```

Comment:
TODO: Do not merge as is ^^^. Only merge after merging https://github.com/algorand/go-algorand-sdk/pull/531 and updating with `go get go-algorand-sdk`.